jujugui/static/gui/src/app/app.js

```diff
@@ -1533,6 +1556,7 @@ YUI.add('juju-gui', function(Y) {
           this.db.machines.size()
         );
         this._renderAddedServices();
+        this._renderSearchResults(req.query.text);
```

Comment:
This no longer exists, I've had to expand the state system and make it idempotent so that we can re-render the UI from the state dispatcher...I can show you in detail